WooCommerce/Classes/Styles/UIColor+MurielColors.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+extension UIColor {
+    /// Get a UIColor from the Muriel color palette
+    ///
+    /// - Parameters:
+    ///   - color: an instance of a MurielColor
+    /// - Returns: UIColor. Red in cases of error
+    class func muriel(color murielColor: MurielColor) -> UIColor {
+        let assetName = murielColor.assetName()
+        let color: UIColor?
+
+        // This is temporary work around as there's a bug in the
+        // GM seed of Xcode 11 which causes loading colors from asset
+        // catalogs to fail (54325712)
+        if #available(iOS 12.0, *) {
+            color = UIColor(named: assetName)
+        } else {
+            color = MurielPalette.color(from: assetName)
```

Comment:
do we still need this since WCiOS is iOS 12+?

Reply:
Good point, we probably don't